apps/web/components/atoms/button/button.tsx

```diff
@@ -22,16 +26,21 @@ export const Button = ({
 	type = 'button',
 	variant,
 	fullWidth = false,
+	className,
```

Comment:
Według naszego design systemu żaden reużywalny przycisk nie powinien posiadać propsa className. Albo dodajemy flage/zachowanie na variant albo robimy to w inny sposób